pyfar/dsp/dsp.py

```diff
@@ -67,8 +67,7 @@ def group_delay(signal, frequencies=None, method='fft'):
     Returns
     -------
     group_delay : numpy array
-        Frequency dependent group delay in samples. The array is flattened if
-        a single channel signal was passed to the function.
+        Frequency dependent group delay in samples.
```

Comment:
```suggestion
        Frequency dependent group delay in samples of shape ``(*signal.cshape, #frequencies)``.
```
maybe something like this? I would not squeeze the dimension of the frequency if its just one.